src/components/Slider/Slider.jsx

```diff
@@ -0,0 +1,186 @@
+import React, { useCallback, useRef } from "react";
+import PropTypes from "prop-types";
+import {
+  getClientPosition,
+  addEventListeners,
+  DIMENSIONS_MAP,
+  POSITIONS_MAP
+} from "./helpers";
+import { classNames } from "../../utils";
+import styles from "./styles.css";
+import SliderHandle, { useSliderHandle } from "./SliderHandle.jsx";
+
+function useSlider(double, disabled, axis, min, max, step, handles) {
+  const container = useRef(null);
+
+  if (double) {
+    handles[0].barrier = useCallback((value) => value <= handles[1].value, [
```

Comment:
Define new const instead of using array everywhere: `const leftHnadle = handle[0]`